src/databrowser.js

```diff
@@ -165,7 +165,7 @@ define([
                 }
                 info = info._parent;
             }
-            vellum.handleDropFinish(target, path);
+            vellum.handleDropFinish(target, path, widget.mug);
```

Comment:
This change doesn't seem right. The thing being dropped here is a data source entity path, which I think is not related to the widget's mug. Can you explain what I'm missing?